File: Service_eHospital_DongNai_A/Implement/Cust_Backup_BsCCHNService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class Cust_Backup_BsCCHNService : BaseService<Cust_Backup_BsCCHN, ICust_Backup_BsCCHNRepository>
    , ICust_Backup_BsCCHNService
    {
    private readonly ICust_Backup_BsCCHNRepository _Cust_Backup_BsCCHNRepository;
    public Cust_Backup_BsCCHNService(ICust_Backup_BsCCHNRepository Cust_Backup_BsCCHNRepository) : base(Cust_Backup_BsCCHNRepository)
    {
    _Cust_Backup_BsCCHNRepository = Cust_Backup_BsCCHNRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A/Cust_Backup_BsCCHN.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Cust_Backup_BsCCHN } from './Cust_Backup_BsCCHN.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Cust_Backup_BsCCHNService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'STT', 'BenhAn_id', 'KhamBenh_id', 'BsKham_Old_id', 'BsKham_New_id', 'ThoiGianTao'];

    List: Cust_Backup_BsCCHN[] | undefined;
    ListFilter: Cust_Backup_BsCCHN[] | undefined;
    FormData!: Cust_Backup_BsCCHN;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Cust_Backup_BsCCHN";
    }
}

